curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md

```diff
@@ -48,7 +48,7 @@ function findGreaterOrEqual(a, b) {
 
 ## Instructions
 <section id='instructions'>
-Use multiple conditional operators in the <code>checkSign</code> function to check if a number is positive, negative or zero. The function should return "positive", "negative" or "zero".
+In the <code>checkSign</code> function, use multiple conditional operators - following the recommended format used in <code>findGreaterOrEqual</code> - to check if a number is positive, negative or zero. The function should return "positive", "negative" or "zero".
```

Comment:
```suggestion
In the <code>checkSign</code> function, use multiple conditional operators - following the recommended format used in <code>findGreaterOrEqual</code> - to check if a number is positive, negative or zero. The function should return <code>"positive"</code>, <code>"negative"</code> or <code>"zero"</code>.
```

Reply:
I'm not sure if it's my responsibility to commit your suggestion or not - this is my first time contributing to anything. But this seems like a good idea and would make the requirements a little clearer.

Reply:
As the author of your PR, you may make any changes you like, including committing suggestions. But, let us both await another member's input.

Reply:
This looks like a good change to me.  @jasnkwcz since you're happy with it, the easiest way to add it is just to click the Commit suggestion button.  For small changes that's the way to go, since it lets the reviewer know that you've implemented exactly what they were asking for.